### ISSUE TITLE
A11y_AzureDataStudioSQLMigration_AzureSQLMigration_Colorcontrast:The link color is similar with the text in the screen and when hovering over the link using mouse pointer there is no visible underline provided for the link.

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/azuredatastudio#15733]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AZUREDATASTUDIOSQLMigration_June2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG1.4.1;#AINotInScope;#Useofcolor

### Environment Details:
Application Name: SQL Azure Data Studio
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure SQL Migration.
2. Connect to any sql server.
3. Tab till Azure SQL migration Tab and press enter.
4. Tab till learn more link and check if there is a visible indication of underline present on the link.

### Actual:
The link color is similar with the text in the screen and when hovering over the link using mouse pointer there is no visible underline provided for the link.
**This issue is observed with links throughout the application.**

### Expected:
When hovering over the links there should be a visible underline provided for the links.

### User Impact:
Users facing difficulty in distinguishing colors will not be able to identify the links.

**Comment by aasimkhan30** [comment]
@alanrenmsft, is this something we want to add?

**Comment by Charles-Gagnon** [comment]
Yes, all links should have the highlight + underline, so if this isn't happening for modelview links or something we should fix that.

**Comment by alanrenmsft** [comment]
none of the links in ADS and VSCode have underline when hovered, it is by design, @Shivamd1 which MAS rule requires this?

**Comment by alanrenmsft** [comment]
just noticed the WCAG 1.4.1 rule, it says we should use a color to indicate a link, which we already did, the color of the link is different from the normal text.

**Comment by alanrenmsft** [comment]
please reopen if this is a new requirement to have underline for links. @Shivamd1

**Comment by Charles-Gagnon** [comment]
That's not true - for example the link button in Notebook toolbar : 

![image](https://user-images.githubusercontent.com/28519865/124309383-7d71a100-db1f-11eb-990f-dd0709d9876f.png)

and then also links in Notebooks themselves : 

![image](https://user-images.githubusercontent.com/28519865/124309478-9d08c980-db1f-11eb-8998-8dc8bd20c615.png)

This is a pretty standard thing, but let's see what the a11y folks say about that being a requirement. 

The links would need to have some sort of focus indicator though so (such as the box you added) so if that isn't happening then we should be fixing that for sure.

**Comment by prabhudevu** [comment]
I have verified the issue in the latest build 1.42.0 issue is fixed hence closing the bug
![15733 fixed](https://user-images.githubusercontent.com/98821792/220149851-d7193f42-d9e5-4c21-919f-59f007f828ea.gif)

**GithubTags:**#Closed;#Regressed:02-20-23;
